src/main/python/janusgraph_python/core/attribute/GeoPredicate/Geo.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .GeoContains import GeoContains
+from .GeoWithin import GeoWithin
+from ...datatypes.GeoShape import GeoShape
+
+
+class Geo(object):
+    """
+    This class defines all the Geo Predicates.
+    """
+
+    contains = GeoContains()
+    within = GeoWithin()
+
+    @staticmethod
+    def geoContains(value):
+        """ The class is used for JanusGraph geoContains predicate.
+
+        GeoContains predicate holds true when one object, is contained by another. Considering the example in docs,
+        the following statement holds true : India (GeoShape) contains (GeoContains) New Delhi (GeoShape)
+
+        Args:
+            value (GeoShape): The GeoShape to query for and return all results which are present inside this GeoShape
```

Comment:
_and return all results which are present inside this GeoShape_ <- This doesn't sound right. A predicate just returns `true` or `false`, indicating whether the currently traversed element fulfils the predicate or not.

Reply:
@FlorianHockmann : Is it? 
Because per my understanding, when I do query something like `g.V().has("name", textContains("something").next()` from gremlin console, the expected output is, the list of vertices which contains the passed string. Its not true or false.

Reply:
Might be a silly question, but will clarify my understanding a lot if you can help :-)

Reply:
That is true, but it's the output of the full traversal. The predicate itself just returns `true` or `false` which is then used to filter the traversed vertices by the `has` step.